lib/features/steps/multipart_request_steps.rb

```diff
@@ -0,0 +1,126 @@
+require 'test/unit'
+require 'open-uri'
+require 'json'
+require 'cgi'
+
+include Test::Unit::Assertions
+
+# @!group Multipart request assertion steps
+
+# Verifies that the request contains multipart form-data
+Then('the request contains multipart form-data and corresponding headers') do
```

Comment:
How about something like this:

```suggestion
Then('the request is valid multipart form-data') do
```